### DISCUSSION TITLE
Is the multitask tagger still predicts supertags?

### DISCUSSION THREAD

**Discussion by oroszgy** [explosion/spaCy#9713] [Help: Model Advice] [answered]
Hi!

I am wondering whether [this description](https://github.com/explosion/spaCy/issues/1057) of the tagger model is still true. I am particularly interested in the supertag part, as [training configuration facilities](https://spacy.io/usage/training) seemingly contradicts with this approach.

Could anyone give me some guidance here?

**Comment by adrianeboyd** [comment, marked as answer]
The tagger doesn't currently predict supertags. There is support for multi-task learning with a shared `tok2vec` or `transformer` component as described here: https://spacy.io/usage/embeddings-transformers#embedding-layers

The provided trained non-trf pipelines have a shared `tok2vec` for all the syntactic components (`tagger`, `morphologizer`, `parser`) and the `ner` component is separate. For the `trf` pipelines, the `transformer` component is shared for all trainable components. A few more details: https://spacy.io/models#design

**Reply by oroszgy** [reply]
I see, thanks for the quick answer!
